api/src/main/resources/liquibase.xml

```diff
@@ -322,4 +322,54 @@
 		</preConditions>
 		<modifyDataType tableName="idgen_seq_id_gen" columnName="prefix" newDataType="varchar(100)"/>
  	</changeSet>
+ 	
+ 	<changeSet id="c7084acb-9cc5-4643-ad6d-6302acc76d69" author="Samuel Male">
+ 		<preConditions onFail="MARK_RAN">
+ 			<not><columnExists tableName="idgen_auto_generation_option" columnName="uuid"/></not>
+ 		</preConditions>
+ 		<comment>
+ 			Adding "uuid" column to the idgen_auto_generation_option table
+ 		</comment>
+ 		
+ 		<addColumn tableName="idgen_auto_generation_option">
+			<column name="uuid" type="UUID" />
```

Comment:
shouldn't this be char(38)

Reply:
Liquibase evaluates it to the target datatype out of the box.

An example is in a similar work done in the earlier Openmrs version: https://github.com/openmrs/openmrs-core/blob/823d9d0ddb38979543da23aa9e4ed2cb54bca064/metadata/model/liquibase-update-to-latest.xml#L878-L882

Reply:
Ah, okay, I did a quick check in the liquibbase update scripts and the first example I found just used "char(38)" so I guess we've been inconsistent... but since we aren't introducing a new inconsistency :) this works for me.

Reply:
@samuelmale this might be true, but you may need to be careful as to whether this depends upon a specific feature in liquibase that might not be in a version used by all versions of core.  Can you check whether the version of liquibase used in OpenMRS 1.9 includes this feature?

Reply:
This feature has been available since the liquibase 1.x series, so, yes it's available. There's a slight mismatch, however: on MySQL liquibase maps UUIDs to `char(36)` rather than `char(38)`. `char(36)` is sufficient to store a UUID; `char(38)` is required for Microsoft's representations of GUIDs (which is surrounded by curly braces).

Reply:
Yes, this works within the liquibase version used by all versions of core; I have also tested this at runtime, out of curiosity, I had to find this:

>To help make scripts database-independent, the following “generic” data types will be converted to the correct database implementation:
>
>BOOLEAN
CURRENCY
UUID
CLOB
BLOB
DATE
DATETIME
TIME
BIGINT

here: https://www.liquibase.org/documentation/column.html

Reply:
>char(36) is sufficient to store a UUID; char(38) is required for Microsoft's representations of GUIDs (which is surrounded by curly braces).

Sure @ibacher, does this mean i change this from `char(38)` to `char(36)` in the Hibernate mapping ?

Reply:
That probably wouldn't hurt.

Reply:
> That probably wouldn't hurt.

Updated :)